### DISCUSSION TITLE
Issue Disabling User via API

### DISCUSSION THREAD

**Discussion by adamleemiller** [drakkan/sftpgo#1451] [General]
First off, THANK YOU! Your application has allowed us to provide a better workflow for third parties to upload large data sets instead of using our previously custom built solution via the browser. We find it easy to use, simple to configure and it just works however, I seem to be having an issue disabling a user via the API that was previously created via the API.

We're strictly using SFTPGO to provide SFTP access to third party vendors that upload large data sets to us that need to be stored using an S3-compatible object storage provider. Unfortunately, when I try to set that status of a user to "0" via the API, I get a weird response: "Validation error: please grant some permissions to this user". The user clearly has some set of permission because the upload works fine. We have a script that verifies the upload has been successfully completed at which point, we disable the user. The user is also set to expire after 48 hours, if it is not disabled first.

The request is to https://{hostname}/api/v2/users/{username} with the body being a JSON object:

`{
    "status": 0
}`

The response I am receiving is below with a 400 response code:

`{
    "error": "Validation error: please grant some permissions to this user",
    "message": ""
}`

Has anyone else ran into this issue? I looked through "Issues" and tried a Google search but nothing has come up. I am going to look through the code to see if I can figure out what it is looking for but figured I would see if anyone else has seen this issue.

Thank you!

**Comment by drakkan** [comment]
https://github.com/drakkan/sftpgo/issues/570

**Reply by adamleemiller** [reply]
Thank you sir, I greatly appreciate it.
